protostar-rust/tests/e2e/running.rs

```diff
@@ -73,3 +77,39 @@ fn run_print_test() {
             Tests: 1 passed, 0 failed
         "#});
 }
+
+#[test]
+fn run_panic_decoding_test() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("tests/data/panic_decoding_test", &["**/*"])
+        .unwrap();
+
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"Collected 4 test(s) and 2 test file(s)
+            Running 0 test(s) from src/lib.cairo
+            Running 4 test(s) from tests/test_panic_decoding.cairo
+            [PASS] test_panic_decoding::test_panic_decoding::test_simple
+            [FAIL] test_panic_decoding::test_panic_decoding::test_panic_decoding
+            
+            Failure data:
+            original value: [123], converted to a string: [{]
+            original value: [6381921], converted to a string: [aaa]
+            original value: [3618502788666131213697322783095070105623107215331596699973092056135872020480]
+            original value: [152]
+            original value: [124], converted to a string: [|]
+            original value: [149]
```

Comment:
I think this could be indented so if you have a lot of tests it's more clearly visible. What do you think?

Reply:
Tried it and I think in console it looks a bit worse, especially a smaller one
![image](https://github.com/software-mansion/protostar/assets/56825108/1f7b1b21-ad60-44d8-908b-47d763e7e8fc)
![image](https://github.com/software-mansion/protostar/assets/56825108/5a1841e1-9fb1-4ef3-8029-405e26bfa43d)
But I don't have a strong opinion on it

Reply:
I think I like the indented one a bit more.